MODULE_TEMPLATE.md

```
@@ -0,0 +1,1058 @@
+- Use `text()` for strings and IDs (D1 is SQLite)
+- Use `integer()` for numbers (store money in cents to avoid floating point issues)
+- Add `.notNull()` for required fields
+- Create Zod schemas with `createInsertSchema` and `createSelectSchema`
+- Add custom validation in Zod schema if needed
+
+---
+
+### Export Schema Centrally
+
+Edit `src/db/schema.ts` and add:
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)